src/jdmtool/skybound.py

```diff
@@ -7,6 +8,15 @@
 class SkyboundException(Exception):
     pass
 
+@dataclass
```

Comment:
I understand if you are not keen on this dataclass, and I can add another `elif` to make the PR less intrusive.